crates/ty/tests/cli.rs

```diff
@@ -368,12 +368,12 @@ fn configuration_rule_severity() -> anyhow::Result<()> {
             for a in range(0, int(y)):
                 x = a
 
-            print(x)  # possibly-unresolved-reference
+            prin(x)  # unresolved-reference
             "#,
     )?;
 
-    // Assert that there's a possibly unresolved reference diagnostic
-    // and that division-by-zero has a severity of error by default.
+    // Assert that there's no possibly unresolved reference diagnostic
+    // but one division-by-zero with a severity of error by default.
```

Comment:
I think this change is wrong/confusing? I would think this should be:
```suggestion
    // Assert that there's a unresolved-reference diagnostic and that
    // division-by-zero has a severity of error by default.
```
The actual test where we change the severities is below.